lix/packages/sdk/src/database/createSchema.ts

```diff
@@ -77,5 +77,39 @@ export async function createSchema(args: { db: Kysely<any> }) {
   BEGIN
     insert or replace into file_internal(id, path, data, metadata) values(OLD.file_id, OLD.path, OLD.data, OLD.metadata);
   END;
+
+
+  -- change discussions 
+
+  CREATE TABLE discussion (
+    -- TODO in inlang i saw we replace uuid_v3 with uuid_v7 any reason we use v4 in lix 
+    id TEXT PRIMARY KEY DEFAULT (uuid_v4())
+  ) strict;
+
+  CREATE TABLE discussion_change_map (
+    discussion_id TEXT NOT NULL,
+    change_id TEXT NOT NULL,
+
+    FOREIGN KEY(discussion_id) REFERENCES discussion(id),
+    
+    -- NOTE this will prevent us from dropping changes 
+    FOREIGN KEY(change_id) REFERENCES change(id)
+  ) strict;
+
+
+
+  CREATE TABLE comment (
```

Comment:
depneding on https://github.com/opral/monorepo/pull/3121#discussion_r1757249621 we can call it `change_discussion_comment` or `discussion_comment`. 

I renamed it to just comment before i saw that discussion id can't be null in the schema you proposed. 

I originally had the mental model of a general comment comment where it could reference a comments and/or something else like a discussion.